### PR TITLE
First version of Configuration service

### PR DESCRIPTION
**Pull request by ulfdawolf**

### CODE REVIEW

File: config/grpcapi.proto [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+message ConfigResponse {
+  string configID = 1;
+  string versionID = 2;
+  string createdAtDate = 3; // RFC_3339 format
+  repeated string targetID = 4;
+  QCMeasurementPointConfig measurementConfig = 5;
+}
+
+message LatestConfigRequest {
+  repeated string targetID = 1;
```
Comment (gurre) [inline config/grpcapi.proto]: `targetIDs`

File: config/grpcapi.proto [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+}
+
+message SetConfigRequest {
+  QCMeasurementPointConfig measurementConfig = 2;
+  repeated string targetID = 1;
```
Comment (gurre) [inline config/grpcapi.proto]: targetIDs

File: config/grpcapi.proto [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+}
+
+enum TargetType {
+  TARGET_GENERIC = 0;
+  TARGET_MEASUREMENT_POINT = 1;
```
Comment (gurre) [inline config/grpcapi.proto]: TARGET_QUICK_COLLECT_MEASUREMENT_POINT

File: config/grpcapi.proto (hunk 1 above)
Comment (bolleb) [reply config/grpcapi.proto]: Done

File: config/grpcapi.proto (hunk 3 above)
Comment (bolleb) [reply config/grpcapi.proto]: Done

File: config/grpcapi.proto [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+}
+
+enum ExternalSensorICP {
+  ICP_UNKNOWN = 0;
+  ICP_REQUIRED_WHEN_IN_USE = 1;
+}
+
+enum ViewSignalType {
+  VIEW_SIGNAL_UNKNOWN = 0;
+  VIEW_SIGNAL_TIME_WAVEFORM_ONLY = 1;
```
[the remark this answers is not in the document]
Comment (bolleb) [reply config/grpcapi.proto]: Done

**Pull request merged**